iaso/api/devices.py

```diff
@@ -21,7 +21,7 @@ class Meta:
     def get_synched_at(obj: Device):
         # updated_at field is better compared to created_at
         instance = Instance.objects.filter(device__id=obj.id).order_by("-updated_at").first()
-        return instance.created_at.timestamp() if instance else None
+        return instance.source_created_at.timestamp() if instance else None
```

Comment:
```suggestion
        return instance.source_created_at.timestamp() if instance and instance.source_created_at else None
```

This can now fail because `source_created_at` is nullable.